.github/ctests.yml

```diff
@@ -0,0 +1,149 @@
+name: CMake
+
+on: [push, pull_request]
+
+env:
+  # Customize the CMake build type here (Release, Debug, RelWithDebInfo, etc.)
+  BUILD_TYPE: Release
+
+jobs:
+  build-library:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    defaults:
+      run:
+        shell: bash -l {0}
+    env:
+      SSPICE_DEBUG: y
```

Comment:
nothing to handle here, but we should probably update the name of this variable as it references the old internal name.